packages/dashboard-core-plugins/src/events/index.ts

```diff
@@ -5,4 +5,6 @@ export { default as IrisGridEvent } from './IrisGridEvent';
 export { default as MarkdownEvent } from './MarkdownEvent';
 export { default as NotebookEvent } from './NotebookEvent';
 export { default as PandasEvent } from './PandasEvent';
-export { default as TabEvent } from './TabEvent';
+
+// Deprecated - use TabEvent from @deephaven/dashboard
+export { type TabEvent } from '@deephaven/dashboard';
```

Comment:
You could do something like this so the deprecation actually shows up in vscode

```ts
import { type TabEvent as TabEventImport } from '@deephaven/dashboard';

/**
 * @deprecated Use TabEvent from `@deephaven/dashboard`
 */
export type TabEvent = typeof TabEventImport; // Not sure why typeof is needed here
```

This will give the deprecation strikethrough in VSCode